package.json

```diff
@@ -25,7 +25,8 @@
     "start": "NODE_ENV=production node build/server.js",
     "stop": "./scripts/killApp.sh",
     "storybook": "start-storybook -p 9001 -s .storybook/static -c .storybook",
-    "test": "npm run test:lint && npm run test:dependencies && npm run test:unit && npm run build -- --hide-modules --colors && run-p --race start amp:validate && run-p --race start test:integration:ci",
+    "test": "npm run build && npm run test:local",
+    "test:local": "npm run test:lint && npm run test:dependencies && npm run test:unit && run-p --race start amp:validate && run-p --race start test:integration:ci",
```

Comment:
I've moved these around in order to preserve the current behaviour of `npm test` for those who rely upon it while allowing the tests to be run without triggering `build` again for CI use.

I don't know if `test:local` is a helpful name, but then to be honest I don't really understand the thinking behind the current split between `test` and `test:ci`.